prime-dotnet-webapi/Controllers/GisController.cs

```diff
@@ -0,0 +1,210 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Services;
+using Prime.Models.Api;
+using Prime.ViewModels.Parties;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
```

Comment:
I think it would make more sense if this route was api/parties/[controller], like the phsa enrolments.